src/relay/README.md

```diff
@@ -88,9 +89,7 @@ function handleResponse(props: AppQueryResponse) {
 export default function App(props) {
   return (
     <QueryRenderer
-      // Following `clientID` helps us to identify who is sending the request and it's
-      // required unless you provide custom `environment` (see below).
-      clientID="unique-client-identification"
+      environment={ ... } // see bellow
```

Comment:
Your most common mistake here 🙈 , `below`

Reply:
omg 😂

Reply:
I fixed it for you ;)

Reply:
Oh, seems you did too 🙈

Reply:
Thanks! :)